src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterOptions.cs

```diff
@@ -65,5 +65,39 @@ public bool SkipValidation
 
         private const int IndentBit = 1;
         private const int SkipValidationBit = 2;
+
+        /// <summary>
+        /// Returns a value that indicates whether this instance is equal to a specified object
+        /// </summary>
+        /// <param name="other">The object to compare with this instance</param>
+        /// <returns>true if other is an instance of <see cref="JsonWriterOptions"/> and equals the value of the instance; otherwise, false</returns>
+        public bool Equals(JsonWriterOptions other)
```

Comment:
For what the options are currently, this change works fine and LGTM.

However, I am concerned about what it would look like once we add `JavaScriptEncoder` (see https://github.com/dotnet/corefx/issues/37192).